src/Login/Login_index.js

```diff
@@ -27,56 +27,54 @@ class App extends Component {
 
 	render() {
 		return (
-			<div className="backbox">
-
-				<div className="welcome-to-the">
-				Welcome to the
-				</div>
-
-				<div className="logo">
-				<img src={require('./Full_Logo.png')} />
-				</div>
-				<MuiThemeProvider >
-				<div className="bottombox">
-					<div>
-						<FlatButton className="no-account-yet-get"
-							label="No account yet? Get setup now >"
-							onClick={(event) => this.onRegister(event)}
-						/>
+			<div className="back">
+				<div className="backbox">
+					<div className="welcome-to-the">
+					Welcome to the
 					</div>
-				</div>
-				</MuiThemeProvider >
-				<div className="loginboxpos">
-				<MuiThemeProvider >
-					<div >
-						<TextField className="loginbox"
-							hintText="Email"
-							floatingLabelText="Email"
-							type="email"
-							onChange = {(event,newValue) => this.setState({email:newValue})}
-						/>
-						<br></br>
-						<TextField className="loginbox"
-							hintText="Password"
-							floatingLabelText="Password"
-							type="password"
-							onChange = {(event,newValue) => this.setState({password:newValue})}
-						/>
+					<div className="logo">
+					<img src={require('./Full_Logo.png')} />
 					</div>
-				</MuiThemeProvider>
-				</div>
-
-				<MuiThemeProvider >
-					<div >
-						<FlatButton className="loginbtn"
-							label="Login"
-							type="submit"
-							onClick={(event) => this.onSubmit(event)}
-						/>
+					<MuiThemeProvider >
+					<div className="bottombox">
+						<div>
+							<FlatButton className="no-account-yet-get"
+								label="No account yet? Get setup now >"
+								onClick={(event) => this.onRegister(event)}
+							/>
+						</div>
 					</div>
-				</MuiThemeProvider>
+					</MuiThemeProvider >
+					<div className="loginboxpos">
+					<MuiThemeProvider >
+						<div >
+							<TextField className="loginbox"
+								hintText="Email"
+								floatingLabelText="Email"
+								type="email"
+								onChange = {(event,newValue) => this.setState({email:newValue})}
+							/>
+							<br></br>
+							<TextField className="loginbox"
+								hintText="Password"
+								floatingLabelText="Password"
+								type="password"
+								onChange = {(event,newValue) => this.setState({password:newValue})}
```

Comment:
Redux!!!!

Reply:
Ek dink ek het Redux in gesit vir validation.

Reply:
Why setstate then?